### PR TITLE
vpu: refactor the vpu plugin readme

### PR DESCRIPTION
**Pull request by alekdu**
Just follow the standard format to fix the vpu plugin readme.
Also added the ubuntu OpenVINO demo job long logs.

Signed-off-by: Alek Du <alek.du@intel.com>

### CODE REVIEW

File: cmd/vpu_plugin/README.md [hunk 1]
```diff
@@ -1,75 +1,240 @@
...
+    * [Deploy by hand](#deploy-by-hand)
+        * [Build the plugin](#build-the-plugin)
+        * [Run the plugin as administrator](#run-the-plugin-as-administrator)
+    * [Verify plugin registration](#verify-plugin-registration)
+    * [Testing the plugin](#testing-the-plugin)
+
+# Introduction
+
+The VPU device plugin supports Intel VCAC-A card:
+(https://www.intel.com/content/dam/www/public/us/en/documents/datasheets/media-analytics-vcac-a-accelerator-card-by-celestica-datasheet.pdf).
```
Comment (mythi) [inline cmd/vpu_plugin/README.md]: there are other cards too, right? perhaps list all of them?

I'd prefer we create links `[Intel VCAC-A](url)`.

File: cmd/vpu_plugin/README.md [hunk 2]
```diff
@@ -1,75 +1,240 @@
...
+The card has:
+- 1 Intel Core i3-7100U processor
+- 12 MyriadX VPUs
+- 8GB DDR4 memory
+
+Note: This device plugin need HDDL daemon service to be running either from native or from a container.
```
Comment (grahamwhaley) [inline cmd/vpu_plugin/README.md]: s/from native/natively/

File: cmd/vpu_plugin/README.md [hunk 3]
```diff
@@ -1,75 +1,240 @@
...
+vcaanode00
+ hddl: 12
 ```
 
-### Test VPU device plugin:
+## Testing the plugin
+
+We can test the plugin is working by deploying the provided example OpenVINO image with HDDL plugin enabled.
+
+1. Build a Docker image with an classification example:
```
Comment (grahamwhaley) [inline cmd/vpu_plugin/README.md]: just a nitpick - I think I changed the numbered lists into titled sections in the other docs, as the numbered lists and thus indented code and words are a bit more painful to maintain in markdown - not a blocker, just a note.

File: cmd/vpu_plugin/README.md [hunk 4]
```diff
@@ -1,75 +1,240 @@
...
+To deploy the vpu plugin as a daemonset, you first need to build a container image for the
+plugin and ensure that is visible to your nodes.
 
-#### Run VPU device plugin as administrator:
-```
-$ sudo $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes/cmd/vpu_plugin/vpu_plugin
-device-plugin start server at: /var/lib/kubelet/device-plugins/vpu.intel.com-hddl.sock
-device-plugin registered
-```
+### Build the plugin image
```
[the remark this answers is not in the document]
Comment (alekdu) [reply cmd/vpu_plugin/README.md]: done

File: cmd/vpu_plugin/README.md (hunk 1 above)
Comment (alekdu) [reply cmd/vpu_plugin/README.md]: done

File: cmd/vpu_plugin/README.md (hunk 2 above)
Comment (alekdu) [reply cmd/vpu_plugin/README.md]: done

File: cmd/vpu_plugin/README.md (hunk 3 above)
Comment (alekdu) [reply cmd/vpu_plugin/README.md]: done

File: cmd/vpu_plugin/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mythi) [reply cmd/vpu_plugin/README.md]: I added my comment in wrong place. The container build should be fine but in section `Build the plugin` where we explain how to build the plugin on the host, the user needs libusb heades.

[the remark this answers is not in the document]
Comment (alekdu) [reply cmd/vpu_plugin/README.md]: Right,  I should capture that :-). Fixed now.

File: cmd/vpu_plugin/README.md [hunk 5]
```diff
@@ -1,75 +1,248 @@
...
+    * [Deploy by hand](#deploy-by-hand)
+        * [Build the plugin](#build-the-plugin)
+        * [Run the plugin as administrator](#run-the-plugin-as-administrator)
+    * [Verify plugin registration](#verify-plugin-registration)
+    * [Testing the plugin](#testing-the-plugin)
+
+# Introduction
+
+The VPU device plugin supports below cards:
+[Intel VCAC-A](https://www.intel.com/content/dam/www/public/us/en/documents/datasheets/media-analytics-vcac-a-accelerator-card-by-celestica-datasheet.pdf).
```
[the remark this answers is not in the document]
Comment (alekdu) [reply cmd/vpu_plugin/README.md]: Oh, good catch.

File: cmd/vpu_plugin/README.md [hunk 6]
```diff
@@ -1,75 +1,248 @@
...
+## Testing the plugin
+
+We can test the plugin is working by deploying the provided example OpenVINO image with HDDL plugin enabled.
+
+### Build a Docker image with an classification example
+
+    ```bash
+    $ cd demo
+    $ ./build-image.sh ubuntu-demo-openvino
+    ...
```
[the remark this answers is not in the document]
Comment (alekdu) [reply cmd/vpu_plugin/README.md]: done. thanks for the tip!

**Pull request merged**